media-api/app/lib/ImageResponse.scala

```diff
@@ -52,6 +52,14 @@ class ImageResponse(config: MediaApiConfig, s3Client: S3Client, usageQuota: Usag
   def hasLeases(image: Image) =
     image.leases.leases.nonEmpty
 
+  def isInPersistedCollection(image: Image): Boolean = {
+    // list of the first element of each collection's `path`, i.e all the root collections
+    val collectionPaths: List[String] = image.collections.map(_.path.head)
```

Comment:
This is using `.head`.

It would be safer to rewrite this as: 

```
val collectionPaths: List[String] = image.collections.flatMap(_.path.headOption)
```

Reply:
👍 https://github.com/guardian/grid/pull/2234